src/code/actions.js

```diff
@@ -1,6 +1,7 @@
 import actionTypes from './action-types';
 import { ITS_ACTORS, ITS_ACTIONS, ITS_TARGETS } from './its-constants';
 import GeneticsUtils from './utilities/genetics-utils';
+import AuthoringUtils from './utilities/authoring-utils';
```

Comment:
I like the idea of collection utility functions for accessing the authoring structure into a single module. I would even go further and have as much of the access as possible mediated through such functions, so that the authoring structure could be changed with minimal effect on other clients.